### ISSUE TITLE
[7.2] cant register and join Arena Skirmish

### ISSUE THREAD

**Issue by chilito** [TrinityCore/TrinityCore#19766]
**Description:**
screenshot https://image.prntscr.com/image/feac8a17c8cc47ddadf8801c95c4f53c.png
then I click on JOIN BATTLE nothing happens  (I'm in group in my second character)

If I turn on debug  by   .debug arena   -nothing happens too.

**Expected behaviour:**

Must be register and teleport to arena zone.

 
**Branch(es):**

7.2 , master

**TC rev. hash/commit:** 
 https://github.com/TrinityCore/TrinityCore/commit/9299e9bde087af7e5d777d5a55a3b79d2b63c48e

**Comment by Eliminationzx** [comment]
confirm

**Comment by codeman8214** [comment]
This is part of the implementation of the honor system that will be added when the honor system is added to master branch, I have been looking into this closely since the pull request was made for the honor system. https://github.com/TrinityCore/TrinityCore/pull/19277

**Comment by tkrokli** [comment]
Related PvP issue (but not identical), regarding `.mod honor` and PvP rewards: #18480

**Comment by GlassFace** [comment]
this is not Related to #18480 and #19277 , **CMSG_BATTLEMASTER_JOIN_SKIRMISH**  this is an unhandled opcode , also needs to update arena system too.
**it is about arena and skirmish arena**

**Comment by tkrokli** [comment]
Fair enough. Do you know enough about it to post any suggestions to how it could be fixed?

**Comment by BoThay** [comment]
no fix for it? :s arenas not working
